### DISCUSSION TITLE
Visual Indication of loss of contact to development server

### DISCUSSION THREAD

**Discussion by Timer** [vercel/next.js#62354] [Ideas]
# Feature request

Next.js should give visual indication when the dev server is down.

## Describe the solution you'd like

Maybe a red bar at the bottom of the screen. Maybe a toast. Maybe a different favicon.

**Comment by jamesmosier** [comment]
Going to look into this!

**Comment by sagar-gavhane** [comment]
Is there any way to programmatically simulate the server is down?

**Comment by samcx** [comment]
Moving this to <picture data-single-emoji=":nextjs:" title=":nextjs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/nextjs/2a85d633ae594556.png" alt=":nextjs:" align="absmiddle"></picture> [discussions](https://github.com/vercel/next.js/discussions) since this is a feature request!
